src/components/common/apiCalls/jiffyApis.tsx

```diff
@@ -53,6 +53,37 @@ export interface GlobalCounts {
     bundleCounter: number;
 }
 
+export interface GetUserOpsHash {
+    verificationGasLimit: string;
```

Comment:
All the value can be null except for 
userOpHash , sender, paymaster, actualGasCost, actualGasPrice and nonce